src/lib.rs

```diff
@@ -538,6 +586,92 @@ impl PublicKey {
         })
     }
 
+     // Partially Blind a message to be signed with public metadata
+     pub fn blind_with_metadata<R: CryptoRng + RngCore>(
+        &self,
+        rng: &mut R,
+        msg: impl AsRef<[u8]>,
+        metadata: &[u8],
+        randomize_message: bool,
+        options: &Options,
+    ) -> Result<BlindingResult, Error> {
+        let msg = msg.as_ref();
+        let modulus_bytes = self.n().to_bytes_be().len();
+        let modulus_bits = modulus_bytes * 8;
+        let metadata_len = metadata.len() as u32; 
+        let metadata_len_bytes = metadata_len.to_le_bytes(); 
```

Comment:
It's idiomatic in Rust to use the same name for a variable when changing only its type, e.g.,
```suggestion
        let metadata_len = metadata.len() as u32; 
        let metadata_len = metadata_len.to_le_bytes(); 
```

Reply:
done